src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -443,7 +443,10 @@ void KeeperStorageSnapshot<Storage>::deserialize(SnapshotDeserializationResult<S
                 node.getChildren().reserve(node.stats.numChildren());
 
         if (ephemeral_owner != 0)
+        {
```

Comment:
I see similar code under ZooKeeperDataReader.cpp::deserializeStorageData. Does that need to be adjusted too?

Reply:
Most likely yes, good catch

Reply:
It's only a non-critical stat while converted creates a storage from ZK data and dumps a snapshot from it.
So it's not needed there, but just in case, I'll the same change.